docs/_src/api/openapi/openapi.json

```diff
@@ -1 +1,739 @@
\ No newline at end of file
+{
+    "openapi": "3.0.2",
+    "info": {
+        "title": "Haystack REST API",
+        "version": "1.1.0"
```

Comment:
Would be nice to automate updating this when we make a new release. For now a left a comment in Notion about manually updating it.

Reply:
It's actually automated! :grin: Check line 21 of `rest_api/application.py`